BookWorm/BookWorm/TableViewCells/ChatCell.m

```diff
@@ -0,0 +1,23 @@
+//
+//  ChatCell.m
+//  BookWorm
+//
+//  Created by Ramya Prabakar on 7/20/22.
+//
+
+#import "ChatCell.h"
+
+@implementation ChatCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    // Initialization code
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+
+    // Configure the view for the selected state
```

Comment:
nit: I assume this is xocde generated comment, maybe we just remove it instead of committing into codebase